data/src/main/java/com/microsoft/azure/kusto/data/ConnectionStringBuilder.java

```diff
@@ -28,7 +27,6 @@
     private ConnectionStringBuilder(String resourceUri) {
         clusterUri = resourceUri;
         username = null;
-        password = null;
```

Comment:
I removed it because it was the least secure (encouraging the user to write their credentials in plaintext), and the current methods should be sufficient.